fxevent/event.go

```diff
@@ -109,6 +110,9 @@ type Supplied struct {
 	// TypeName is the name of the type of value that was added.
 	TypeName string
 
+	// StackTrace is the stack trace of the call to Supply.
+	StackTrace string
```

Comment:
Somewhat strange to represent StackTrace as a single string - this is inconvenient since user has to parse the string to get each stack as a separate string. A more commonly used approach here is to use []string.

Should be fairly straightforward to modify this since fxreflect.Stack already keeps track of each stack frame as a separate entity, iirc.

Reply:
Good call, made a change to represent as a list of strings, each being one frame.